src/lib/missing-transition-target.js

```diff
@@ -15,17 +15,11 @@ module.exports = (definition) => {
 
   // check if all states are reachable
   const unreachable = machineStates.filter(state => reachableStates.indexOf(state) === -1)
-    .map(state => ({
-      'Error code': 'MISSING_TRANSITION_TARGET',
-      Message: `State ${state} is not reachable`,
-    }));
+    .map(state => `State ${state} is not reachable`);
```

Comment:
This looks like a unnecessary breaking change, couldn't you keep it as before and do `missingTransitionTargetErrors.map(e => e.Message).join(separator)` in the validator instead ?
Or even ```missingTransitionTargetErrors.map(e => `${e.Code}: ${e.Message}`).join(separator)```